### DISCUSSION TITLE
window-height setting only works if it's more than 22

### DISCUSSION THREAD

**Discussion by sidesteps** [ghostty-org/ghostty#3779] [Q&A]
On linux xfce with standard composing windows manager (wezterm, xfce-terminal respects those settings no problem)

```
window-width=196
window-height=16
```

but ghostty always starts at height 22.

```
$tput lines 
->22
```

If I set window-height to more than 22, then the setting is respected. 22 seems be half of my FHD screen height. ghostty seems to stick to that.

Also you can see how ghostty window is opened with it's default width/height and then resized. 
It happens fast but still noticeable. Other terminals seem to open their window with width/height set from config already.

**Comment by Tomcatt** [comment]
I have 2 computers that I can resize the window.  The 3rd only the height can be adjusted.  All three run EndeavourOS.

**Reply by pluiedev** [reply]
Please do not revive stale discussions — we need more information about your system and setup which may be different from the OP's. Open a new discussion under the "Issue Triage" category.
